API.en.md

```diff
@@ -0,0 +1,126 @@
+
+
+## InfoClient
+
+This class is designed to actively receive information from the Telegram server and has the following methods:  
+  
+`login (AuthKey $authKey [, Proxy $proxy [, callable $cb]])` - authorization of the client on the Telegram server.  
+* *$authKey* - authorization key received after registering the phone number on the Telegram server.  
+* *$proxy* - proxy server abstraction  
+* *$cb* - callback function to be executed after authorization.  
+  
+`isLoggedIn ()` - client authorization check  
+  
+`pollMessage ()` - awaiting receipt of messages from the Telegram server  
+  
+`getChatMembers (int $id, callable $onComplete)` - getting chat participants.  
+* *$id* - chat id  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with requested information  
+  
+`getChannelMembers (int $id, int $accessHash, callable $onComplete)` - getting channel members  
+* *$id* - channel id  
+* *$accessHash* - hash coming from the server in the ‘access_hash’ channel information field  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with requested information  
+  
+`getFullChannel (int $id, int $accessHash, callable $onComplete)` - get channel info with all settings (notifications, permissions, photo, etc.)  
+  
+`getChatMessages (int $id, int $limit, int $since, int $lastId, callable $onComplete)` - receive messages from the chat with the ability to filter by time and quantity.  
```

Comment:
`from chat, filtered by time sent; $limit, max number of messages expected to be received, is <= 100`